images/ose-containernetworking-plugins.yml

```diff
@@ -15,6 +15,7 @@ enabled_repos:
 - rhel-server-optional-rpms
 from:
   builder:
+  - stream: rhel-8-golang
```

Comment:
@dougbtv should this be rhel-8-golang-1.12?

Reply:
This value needs to map to values that are present in the /streams.yml, e.g. https://github.com/openshift/ocp-build-data/blob/openshift-4.3/streams.yml

Reply:
ack, noted.